client/src/app/components/rules/rules.component.ts

```diff
@@ -69,19 +72,19 @@ export class RulesComponent implements OnInit {
   }
 
   // eslint-disable-next-line require-jsdoc
-  ngOnInit(): void {}
+  ngOnInit(): void { }
 
   // eslint-disable-next-line require-jsdoc
   ngAfterViewInit() {
     this.dataSource.paginator = this.paginator;
   }
 
   /**
-   * Fetch all rules.
+   * Fetch all rules for logged in user.
    */
   loadRules() {
     this.http
-      .get<Array<Rule>>(`${environment.apiUrl}/rules`)
+      .get<Array<Rule>>(`${environment.apiUrl}/rules/user/${this.user?.id}`)
```

Comment:
Why are we explicitly passing the user ID in the URL? The server should obtain this information from the authentication the user provides.

Reply:
We previously discussed the need of passing user id to the api endpoint. 
This is required by design.